### ISSUE TITLE
Add editorconfig file for JetBrains Fleet

### ISSUE THREAD

**Issue by LaoshuBaby** [osmlab/name-suggestion-index#7232]
I used to editing NSI with VSCode, and I have a try of Fleet today, but it keep trying to add 4 space indent when edit a line.

So you'll see diff like this: https://github.com/osmlab/name-suggestion-index/commit/32b369394c2276291a3e6a48f9d0e1ecf6411da2#diff-c6acdcf2dc15f704a05c5d6cf7806d3b5dacb2901545101ac35f90fb59d58b72 when some one run fotmatter.

![图片](https://user-images.githubusercontent.com/42690037/197074343-6fa4c2d2-47c2-4b56-98b2-11a86880e86d.png)

**Comment by bhousel** [comment]
Sure, I'm ok if you want to add one.

First, have you tried installing this plugin:
https://www.jetbrains.com/help/idea/editorconfig.html

It might just work with the `.editorconfig` file existing in the repo already.

**Comment by LaoshuBaby** [comment]
> First, have you tried installing this plugin:

Yes, for JB's old editor this works, but newly released Fleet don't support plugin……

<img src="https://user-images.githubusercontent.com/42690037/197360540-6426946c-be9c-4d64-b3e0-30597ede3547.png" width=500px/>

And I only found editorconfig support for C# in its [documents](https://www.jetbrains.com/help/fleet/getting-started.html?q=editorconfig&s=full), and don't found 

<img src="https://user-images.githubusercontent.com/42690037/197360628-68b96234-49f9-4a8e-820f-c8da81b1871b.png" width=500px/>

Yesterday I found a `.fleet` folder and a empty `settings.json`(it was hidden in Fleet), **maybe I can [wrote a editor specific config](https://www.jetbrains.com/help/fleet/settings.html#workspace-settings) like what we do in `.vscode`**, so I assigned myself.

![图片](https://user-images.githubusercontent.com/42690037/197360787-ed493754-3743-4a03-912f-30017bbb471f.png)
